contracts/schemes/ContinuousLockingToken4Reputation.sol

```diff
@@ -67,18 +67,19 @@ contract ContinuousLocking4Reputation is Agreement {
      * @param _avatar the avatar to mint reputation from
      * @param _reputationReward the reputation reward per locking batch that this contract will reward
      *        for the token locking
-     * @param _startTime locking period start time
-     * @param _batchTime batch time (e.g 30 days).
-     * @param _redeemEnableTime redeem enable time .
+     * [This is wrong in two ways: (1) this value is not used in the caluclation of the reputation reward - which is caculated using constA, and (b) the repuation reward is not constant]
+     * @param _startTime locking period start time, in seconds since epoch
+     * @param _batchTime batch time (in seconds)
+     * @param _redeemEnableTime redeem enable time, in seconds since epoch.
      *        redeem reputation can be done after this time.
-     * @param _maxLockingBatches - maximum number of locking batches (in _batchTime units)
-     * @param _repRewardConstA - reputation allocation per batch is calculated by :
-     *   _repRewardConstA * (_repRewardConstB ** batchIndex)
-     * @param _repRewardConstB - reputation allocation per batch is calculated by :
-     *   _repRewardConstA * (_repRewardConstB ** batchIndex)
-     * @param _batchesIndexCap  the max batch index which allows to lock in . this value capped by BATCHES_HARDCAP
+     * @param _maxLockingBatches - maximum number of locking batches that a user can lock her tokens for [JG: not really, see omment on extend function]
```

Comment:
maximum number of locking batches that user can lock or extend in one transaction.